assets/src/blocks/Splittwocolumns/SplittwocolumnsInPlaceEdit.js

```diff
@@ -0,0 +1,111 @@
+import { Fragment } from '@wordpress/element';
+import { RichText } from '@wordpress/block-editor';
+import { debounce } from 'lodash';
+
+const { __ } = wp.i18n;
+
+/**
+ * WYSIWYG in-place editor 
+ */
+export const SplittwocolumnsInPlaceEdit = ({attributes, charLimit, setAttributes}) => {
+  const toAttribute = attributeName => value => setAttributes({
+    [attributeName]: value
+  });
+  const debounceToAttribute = attributeName => debounce(
+    toAttribute(attributeName), 400
+  );
+
+  return (
+    <Fragment>
+    <section className="block-wide split-two-column">
+      <div className="split-two-column-item item--left">
+        {attributes.issue_image_src &&
```

Comment:
Maybe you could destructure `attributes` at the top of the component instead of always repeating `attributes.`?

Reply:
Same in SplittwocolumnsSettings.js